var/spack/repos/builtin/packages/py-mpi4py/package.py

```diff
@@ -19,6 +19,7 @@ class PyMpi4py(PythonPackage):
     license("BSD-2-Clause")
```

Comment:
In mpi4py-4.0.0, the license was changed to BSD-3-Clause

```suggestion
    license("BSD-3-Clause")
```

Reply:
Added now